Project/src/MakeCall/CallCard.js

```diff
@@ -516,6 +544,16 @@ export default class CallCard extends React.Component {
         }
     }
 
+    async handleRaiseHand() {
+        try {
+            this.state.isHandRaised ?
+                this.raiseHandFeature.lowerHand():
+                this.raiseHandFeature.raiseHand();
+                this.setState({isHandRaised: utils.isParticipantHandRaised(this.identifier, this.raiseHandFeature.getRaisedHands())})
```

Comment:
indentation here is out, it reads as if this line is part of the ternary expression